slc_ci_setup/doctor.py

```diff
@@ -0,0 +1,90 @@
+"""
+The doctor module provides functionality to check the health of the `slc-ci-setup`
+package and also provide help to find potential fixes.
+"""
+import shlex
+import subprocess
+import sys
+from enum import Enum
+from typing import Iterator
+
+SUPPORTED_PLATFORMS = ["linux", "darwin"]
+
```

Comment:
Use error reporting python

Reply:
error-reporting-python still has Python 3.6 as minimum version. I would prefer to use it as soon as it upgrades to Python 3.8.

Reply:
Done